Code/skyrim_ui/src/app/components/settings/settings.component.html

```diff
@@ -37,19 +37,19 @@ <h2>{{ 'COMPONENT.SETTINGS.UI' | transloco }}</h2>
       <h2>{{ 'COMPONENT.SETTINGS.PARTY' | transloco }}</h2>
       <div class="flex-item">
         <label for="showParty">{{ 'COMPONENT.SETTINGS.SHOW_PARTY' | transloco }}</label>
-        <app-checkbox
+        <input type="checkbox"
```

Comment:
put it in a new line for consistency
```suggestion
        <input
          type="checkbox"
```